src/app/helpers/functions_helper.rb

```diff
@@ -1,3 +1,6 @@
+require 'kramdown'
+require 'kramdown-parser-gfm'
```

Comment:
[Q] why do we need to require it manually? didn't bundler already required them?

Reply:
I didn't know that I thought I had to require them manually :D